src/dynamicprompts/samplers/combinatorial.py

```diff
@@ -32,136 +30,113 @@ def _dedupe(arr: list[str]) -> tuple[str, ...]:
 
 
 def _combo_to_prompt(
```

Comment:
The name of this function is misleading and the generic return type hides the function's purpose. It takes in a combination of commands and samples each command, returning a list of prompt tokens. How about changing it to something like:
 `def _sample_command_combinations(...) -> Iterable[prompt_tokens]`